torch/onnx/symbolic.py

```diff
@@ -446,6 +446,15 @@ def leaky_relu(g, input, negative_slope, inplace=False):
     return g.op("LeakyRelu", input, alpha_f=_scalar(negative_slope))
 
 
+def linear(g, input, weight, bias=None):
+    if input.dim() == 2 and bias is not None:
+        return addmm(bias, input, weight)
+    output = g.op("MatMul", input, weight.t())
```

Comment:
I don't think `weight.t()` here is correct,  I think it's correct to call the `t()` function defined in this file to emit the proper ONNX nodes to transpose this weight Value

Reply:
thx